src/test/graphql/queries/suggested-tags.integration.ts

```diff
@@ -127,8 +126,8 @@ describe('tags query tests - happy path', () => {
       const tags = res.body.data?._entities[0].savedItemById.suggestedTags;
       expect(tags.length).to.equal(3);
       expect(tags[0].name).to.equal('adventure');
-      expect(tags[1].name).to.equal('thriller');
-      expect(tags[2].name).to.equal('romance');
+      expect(tags[1].name).to.equal('romance');
```

Comment:
coz we removed `order by updatedAt` for suggestedTags, the order is different but the top 3 values are same